### DISCUSSION TITLE
How to use tabler icon ?

### DISCUSSION THREAD

**Discussion by thomas-louvigne** [CaiJimmy/hugo-theme-stack#921] [Q&A]
The documentation says to use tabler icon, but when i go to the /assets/icon/ folder i see only 21 icons.
When i go to the tabler icon website, they talk about more than 3000 icons.
Did i miss something ?

**Comment by Lovirr** [comment]
add a new file, for example "example.svg", and the paste the content in it
<img width="1517" alt="image" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/762afff4-ac1a-48cf-a8de-2ed17f9d4c12">

**Comment by thomas-louvigne** [comment]
Yeah, thank you !
I add a folder `/assets/icons` in my project and NOT in the theme.
The 21 icons are in the theme folder. 
I think the doc is little bit confusing.

Thanks for all
